src/appServiceNodes.ts

```diff
@@ -88,6 +89,20 @@ export class AppServiceNode extends NodeBase {
         }
     }
 
+    async getChildren(azureAccount: AzureAccountWrapper): Promise<NodeBase[]> {
+        if (azureAccount.signInStatus !== 'LoggedIn') {
```

Comment:
Will the status ever be something other than 'LoggedIn' here, since the parent node also requires sign-in?

Reply:
Can signInStatus change after logging in? Maybe during a disconnect or something?  Is that something that we have to worry about?